hack/test-go.sh

```diff
@@ -158,10 +158,16 @@ done
 gotest_flags+=" $*"
 
 # Determine packages to test
+godeps_package_prefix="Godeps/_workspace/src/"
+godeps_package_prefix_length=${#godeps_package_prefix}
 test_packages=
 if [[ -n "${package_args}" ]]; then
     for package in ${package_args}; do
-        test_packages="${test_packages} ${OS_GO_PACKAGE}/${package}"
+        if [[ "${package}" == "${godeps_package_prefix}"* ]]; then
+            test_packages="${test_packages} ${package:$godeps_package_prefix_length}"
```

Comment:
Then we should update the way we generate Kubernetes package names below as well.